hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConsts.java

```diff
@@ -204,6 +204,16 @@ public static Versioning getVersioning(boolean versioning) {
    */
   public static final long MAX_QUOTA_IN_BYTES = 1024L * 1024 * TB;
 
+  /**
+   * Quota value.
+   */
+  public static final long QUOTA_COUNT_RESET = -1;
```

Comment:
Could you please update the comment to "Quota of bucket counts" and add the description of meaning of "-1" here ?